ecosystem/sep-0024.md

```diff
@@ -190,6 +191,72 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 5. `Anchor` detects the trust line creation in the `Account`. If the asset is `AUTH_REQUIRED`, `Anchor` approves the new trust line.
 6. `Anchor` proceeds with the deposit flow.
 
+##### Claimable Balances
+
+[Claimable Balances](https://www.stellar.org/blog/protocol-14-improvements) allows payments to accounts that do not have a trust-line for the asset sent.
+
+**Wallets** poll for their users' received Claimable Balances so that users can "claim" them.
+
+**Anchors** can optionally use this functionality to send SEP-24 deposit transactions to Stellar accounts that don't yet have a trust-line to the asset they have requested.
```

Comment:
I think we should emphasize that supporting claimable balances is currently _optional_ for anchors but _mandatory_ for wallets. Wallets reading this need to infer that some of their partner anchors may implement claimable balances, and therefore need to support it as well, but it's not super clear.